### PR TITLE
Remove redundant logging (and log text generation)

### PR DESCRIPTION
**Pull request by jonmv**
@hmusum please review and merge.

### CODE REVIEW

File: zkfacade/src/main/java/com/yahoo/vespa/curator/CuratorCompletionWaiter.java [hunk 1]
```diff
@@ -57,8 +57,10 @@ public void awaitCompletion(Duration timeout) {
         List<String> respondents;
         do {
             respondents = curator.framework().getChildren().forPath(barrierPath);
-            log.log(Level.FINE, respondents.size() + "/" + curator.zooKeeperEnsembleCount() + " responded: " +
-                                respondents + ", all participants: " + curator.zooKeeperEnsembleConnectionSpec());
+            if (log.isLoggable(Level.FINE)) {
```
Comment (hmusum) [inline zkfacade/src/main/java/com/yahoo/vespa/curator/CuratorCompletionWaiter.java:60]: You can use `log.log(Level.FINE, () -> `instead

**Review by hmusum — APPROVED** [review]
One minor comment

File: zkfacade/src/main/java/com/yahoo/vespa/curator/CuratorCompletionWaiter.java (hunk 1 above)
Comment (jonmv) [reply zkfacade/src/main/java/com/yahoo/vespa/curator/CuratorCompletionWaiter.java:60]: Yes, but in this particular case one of the arguments is non-final, and making it final is much more work :)

Comment (hmusum) [reply zkfacade/src/main/java/com/yahoo/vespa/curator/CuratorCompletionWaiter.java:60]: Yes, right, that's a valid case for sure

**Pull request merged**